### PR TITLE
Fix wording in charter page

### PR DESCRIPTION
**Pull request by vannyle**
**Describe what changes this pull request brings**
This pull request brings the Charter page.

**Additional changes**
- Prevented the horizontal scroll by changing the absolute positioning of the submenu in the header navigation.

**Steps to steps**
1. Open [the page](https://deploy-preview-171--ebpfio.netlify.app/charter)
2. Confirm that everything looks and works as expected

### CODE REVIEW

File: src/pages/charter.js [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+      <h2>7) Good Standing</h2>
+      <ol>
+        <li>The Linux Foundation’s Good Standing Policy is available at <a href="https://www.linuxfoundation.org/good-standing-policy" target="_blank" rel="noopener noreferrer">https://www.linuxfoundation.org/good-standing-policy</a> and will apply to Members of this Directed Fund.</li>
+      </ol>
+      <h2>8) Trademarks</h2>
+      <ol>
+        <li>Any trademarks relating to the Directed Fund or the Technical Projects<a href="#clarification-1"><sup>1</sup></a> , including
+        without limitation any mark relating to any conformance program, must be transferred to
+        and held by LF Projects, LLC or the Linux Foundation and available for use pursuant to
+        LF Projects, LLC’s trademark usage policy, available at <a href="www.lfprojects.org/trademarks/" target="_blank" rel="noopener noreferrer">www.lfprojects.org/trademarks/</a>.</li>
```
Comment (qmonnet) [inline src/pages/charter.js:249]: This link seems broken in the preview, it misses the `https://` prefix.

File: src/pages/charter.js [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+      </ol>
+      <h2>9) Antitrust Guidelines</h2>
+      <ol>
+        <li>All Members must abide by The Linux Foundation’s Antitrust Policy available at <a href="http://www.linuxfoundation.org/antitrust-policy" target="_blank" rel="noopener noreferrer">http://www.linuxfoundation.org/antitrust-policy</a>.</li>
```
Comment (qmonnet) [inline src/pages/charter.js:253]:
```suggestion
        <li>All Members must abide by The Linux Foundation’s Antitrust Policy available at <a href="https://www.linuxfoundation.org/antitrust-policy" target="_blank" rel="noopener noreferrer">https://www.linuxfoundation.org/antitrust-policy</a>.</li>
```

File: src/pages/charter.js [hunk 3]
```diff
@@ -0,0 +1,309 @@
+import Layout from "../layouts";
+import React from 'react'
+import { Helmet } from "react-helmet";
+
+const pageMetaTitle = 'The eBPF Foundation Charter'
+const pageMetaDescription = 'eBPF is a revolutionary technology that can run sandboxed programs in the Linux kernel without changing kernel source code or loading a kernel module.'
```
Comment (qmonnet) [inline src/pages/charter.js:6]: This should be updated to reflect the current page content?

File: src/pages/charter.js [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+const CharterPage = ({ location: {pathname} }) => {
+  return <Layout path={pathname}>
+    <div className="page-wrapper page-charter">
+    <Helmet
+        title={pageMetaTitle}
+
+        meta={[
+          {name: "keywords", content: "ebpf, bpf, contribute, getting started, community"},
```
Comment (qmonnet) [inline src/pages/charter.js:15]: These should also be updated, `contribute` and `getting started` in particular don't sound relevant for this page.

File: src/pages/charter.js [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+          {name: "type", property: "og:type", content: "website"},
+          {name: "url", property: "og:url", content: "https://ebpf.io/contribute/"},
```
Comment (qmonnet) [inline src/pages/charter.js:17]: This field doesn't look right, please double-check. Can you also make sure that all relevant metadata fields are updated for the new page?

**Review by qmonnet — CHANGES_REQUESTED** [review]

File: src/pages/charter.js (hunk 1 above)
Comment (vannyle) [reply src/pages/charter.js]: Thanks for pointing that out. I will update the links.

File: src/pages/charter.js (hunk 4 above)
Comment (vannyle) [reply src/pages/charter.js]: The keywords are updated, replacing  `contribute` and `getting started` with `charter`.

**Review by dthaler — APPROVED** [review]

File: src/pages/charter.js [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+      <ol>
+        <li>The Linux Foundation will have custody of and final authority over the usage of any fees,
+        funds and other cash receipts.</li>
+        <li>A General & Administrative (G&A) fee will be applied by The Linux Foundation to
+        funds raised to cover membership records, finance, accounting, and human resources
+        operations. The G&A fee will be 9% of the Directed Fund’s first $1,000,000 of gross
+        receipts each year and 6% of the Directed Fund’s gross receipts each year over
+        $1,000,000.</li>
+      </ol>
+      <h2>12) General Rules and Operations. <span>The Directed Fund activities must:</span></h2>
```
Comment (dthaler) [inline src/pages/charter.js:285]:
```suggestion
      <h2>12) General Rules and Operations.</h2>
         <span>The Directed Fund activities must:</span>
```

**Comment by tgraf** [comment]
LGTM. Let's wait for final review. It matches the correct version of the charter now.

**Comment by dthaler** [comment]
@borkmann any objection to merging?

**Pull request merged**